.github/workflows/release_begin.yaml

```diff
@@ -81,27 +81,36 @@ jobs:
           if-no-files-found: error
   release:
     if: ${{ github.repository_owner == 'bazelbuild' }}
-    needs: build
+    needs: crate_universe_builds
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
-          # Save the git credentials so we can commit back to the repo
-          persist-credentials: true
+          ref: main
+      - run: |
+          # Write new releases file
+          cat << EOF > ${{ github.workspace }}/version.bzl
+          """The version of rules_rust."""
```

Comment:
Can you add a note to this generated file that merging any changes to this file will automatically trigger a release, and that it should never be manually edited? That's non-obvious at the moment from the file, and feels important, lest someone accidentally release by reformatting or adding a comment or similar.

Reply:
Can I have a dumb question? Why do changes to this file trigger the release workflow?

Reply:
I was looking for an automated way to know that a release just occurred so we can create a new release and upload all the built artifacts to it. Otherwise it either becomes a two manual step process or a release is made before the state of the repo is updated to reflect that release.

Reply:
Got it, thank you. It's a smart solution. If it turns out it is too smart for us we can change it in the future :) Thank you!

Reply:
I'd +1 Daniel's suggestion to add a comment into the file so uninitiated people understand that they should not modify the file by hand.

Reply:
> > (Adding the warning text into version.bzl is uncontroversial even in this PR, wdyt?)
> 
> Sorry, what text? I think I missed something 😅
> 
> I'm happy to still make updates but I don't want to go down a rabbit hole of generating docs and release notes. I'd much rather do that as a separate PR.

Hopefully this link really points where I hope it does: https://github.com/bazelbuild/rules_rust/pull/713#discussion_r622918928

Reply:
I've moved the template for `version.bzl` out into it's own file and it now has a note about the impacts of updating it.